spot-client/src/common/app-state/spot-tv/actions.js

```diff
@@ -53,8 +53,7 @@ export function setSpotTVState(newState) {
  */
 export function storePhoneNumberFromInvites(invites) {
     return dispatch => {
-        const phoneInvites = invites && invites.filter(invite => invite.type === 'phone');
-        const phone = phoneInvites && phoneInvites[0] && phoneInvites[0].number;
+        const phone = invites?.filter(invite => invite.type === 'phone')?.[0]?.number;
```

Comment:
This could be done as a fine instead for now.

Reply:
oh okay